### PR TITLE
Network map, non-citizen upcharge, pretty links, simplified profile editing

### PR DESCRIPTION
**Pull request by namedotget**
- Removed unused files
- Added network/moon maps
- Added map link to network page
- Refactor
- Added non-citizen marketplace upcharge
- Added pretty links for citizens and teams
- Simplified profile editing

### CODE REVIEW

File: ui/components/layout/Sidebar/Navigation.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const navigation = [
       },
       { name: 'Events', href: '/events' },
       { name: 'Analytics', href: '/analytics' },
-      { name: 'Current Projects', href: '/current-projects'}
+      // { name: 'Current Projects', href: '/current-projects'}
```
Comment (jaderiverstokes) [inline ui/components/layout/Sidebar/Navigation.ts:40]: did you mean to leave this in?

File: ui/components/layout/Sidebar/Navigation.ts [hunk 2]
```diff
@@ -37,7 +37,6 @@ export const navigation = [
       },
       { name: 'Events', href: '/events' },
       { name: 'Analytics', href: '/analytics' },
-      { name: 'Current Projects', href: '/current-projects'}
```
Comment (jaderiverstokes) [inline ui/components/layout/Sidebar/Navigation.ts:40]: double checking that you meant to remove this?

File: ui/lib/citizen/useCitizenEmail.tsx [hunk 3]
```diff
@@ -20,23 +21,34 @@ export default function useCitizenEmail(nft: any) {
...
+          process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_SHORT_FORM_ID as string,
+          formResponseId,
+          'JEiG9XCW6M73',
+          accessToken
+        )
+
+        const citizenEmailFormEmail = await fetchEmail(
+          process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_EMAIL_FORM_ID as string,
+          formResponseId,
+          'Z3IMkpvJUfdl',
```
Comment (jaderiverstokes) [inline ui/lib/citizen/useCitizenEmail.tsx:42]: do you want to move this to const file?

Comment (jaderiverstokes) [reply ui/lib/citizen/useCitizenEmail.tsx:42]: also a few others in useTeamEmail

File: ui/pages/api/marketplace/marketplace-purchase.ts [hunk 4]
```diff
@@ -51,7 +69,55 @@ async function handler(req: any, res: any) {
       return res.status(400).send({ message: 'Bad request' })
     }
 
-    const { teamEmail } = JSON.parse(data)
+    const { teamEmail, txHash, isCitizen, recipient, value, decimals } =
```
Comment (jaderiverstokes) [inline ui/pages/api/marketplace/marketplace-purchase.ts:72]: do we want to validate the price here for citizen discounts?

**Review by jaderiverstokes — APPROVED** [review]

**Review by pmoncada — CHANGES_REQUESTED** [review]
Colin - Sorry but I'm going to ask you to dis-aggregate these PRs, 78 files changes is too much to review all at once, and there will likely be bugs that are overlooked.

I'd like you to separate these PRs by feature, and include tests for each new feature introduced.

File: ui/lib/citizen/useCitizenEmail.tsx (hunk 3 above)
Comment (namedotget) [reply ui/lib/citizen/useCitizenEmail.tsx:42]: > do you want to move this to const file?

they should never change and won't be used anywhere else

**Pull request closed without merging**